app/src/main/java/com/example/learn09082021v001/RetrofitAndroidKotlin/APIClient.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.learn09082021v001.RetrofitAndroidKotlin
```

Comment:
put all of class related to retrofit in the new package, can call it is `network`

Reply:
fix done